packages/utils/src/utils/split-utxos.ts

```diff
@@ -0,0 +1,23 @@
+import { ErrorCode, FuelError } from '@fuel-ts/errors';
+import type { AbstractAddress } from '@fuel-ts/interfaces';
+import type { BN } from '@fuel-ts/math';
+
+export function splitUTXOs(
+  balance: BN,
+  amount: BN,
+  assetId: string,
+  destination: AbstractAddress,
```

Comment:
I believe this helper would benefit from some TSDoc

Reply:
0bad71912